### DISCUSSION TITLE
Filter cards in Organization Projects by OR

### DISCUSSION THREAD

**Discussion by lydiayliu** [community/community#19455] [Projects and Issues]
It is very opaque to me whether this function is supported, sounds like it is supported in issues but not in Projects? If I enter two things in the `Filter by keyword or by field`, separated by `,`, the operation seems to be AND not OR

**Comment by tom-os** [comment]
This is an issue for us also, it seems that the filtering syntax supported in Projects (and Issues for that matter) is very limited.

An 'or' operator would be a great start.
Other very useful additions would be logical grouping '()', and a not operator '!'.

Including these few extra things would massively improve using Projects.
